src/test/datascience/notebook/updateNotebookMetadata.unit.test.ts

```diff
@@ -133,18 +133,23 @@ suite('UpdateNotebookMetadata', () => {
             orig_nbformat: 4,
             kernelspec: { display_name: 'Python Default', language: 'python', name: 'python3' },
             language_info: { name: 'python', version: '3.6.0' },
-            interpreter: {
```

Comment:
it would good to have a test for the old metadata too since that will be something we see a lot for a while. Should the metadata be updated? Or will there be stale information left behind?

Reply:
Good point